pkg/daemon/update.go

```diff
@@ -672,13 +680,37 @@ func (dn *Daemon) writeFiles(files []igntypes.File) error {
 				return fmt.Errorf("failed to retrieve file ownership for file %q: %v", file.Path, err)
 			}
 		}
+		if err := createOrigFile(file.Path); err != nil {
```

Comment:
We should *only* do this if the old config doesn't have the file.  Otherwise, in the case of applying configs A, B, C where A and B both modify a file and C doesn't, we'll end up with the content of config A's file, not the true default right?

(Or maybe simplest to implement, only create the backup file if it doesn't already exist)

Reply:
fixed